### ISSUE TITLE
Algorithmically generate folders for all dates within range

### ISSUE THREAD

**Issue by philshem** [co2birthdate/website#1]
This means that the website will have folders

    yyyy/mm/dd/

that are generated in a smart calendar way, e.g. `1980/02/29` should exist

Seems it's solved several ways: https://stackoverflow.com/questions/25701265/how-to-generate-a-list-of-all-dates-in-a-range-using-the-tools-available-in-bash

I'd suggest using 1900/01/01 as a starting date, although if the repo doesn't get too bulky, we can go back to 1800 or something, just for fun.

**Comment by jalbertbowden** [comment]
ok. well i really screwed something up trying to whip this up really quick.
[create_dir.py](https://github.com/co2birthdate/website/blob/master/assets/py/create-dir.py) is a really bad attempt, and its only filling out the dates for January. Again, crude attempt to get something going.
I cannot get the directory of the directories to save.
It could be that we are over limits, but I think I did something wrong with Git.
When I delete the directory from the repository, it executes. then doing an `ls` still shows the data directory.
I have tried a number of methods...fail.

Last resort was to nuke the repo and drop it all in fresh, but I don't think that I have admin for that.
I'm going to save all of the issues here, but then would you mind deleting this?
Maybe we can walk through a fix on my end, but this is always so much easier imo.

**Comment by jalbertbowden** [comment]
still having issues in my repo. i wiped my local repo, forked it to my account, and then cloned it.
started seeing some of the same issues from earlier, and I'm still not even sure what is happening. I can explain in detail but I wanted to test a few things out, so I created a new repository under my account:
https://github.com/jalbertbowden/website-test
I went in clean and created two new dirs locally, and went to add/commit, but they were not showing up as new activity in Git. git status said nothing, up to date with master. `ls` clearly showed the directories existed...

I created `readme.md` for each dir, the tried git status, viola, both new dirs showed up in Git.

So I took your python script and added a new function to create an empty readme in each directory.
Its not ideal, and we should run it again at some point, at the very least populating it, but more likely to generate the html or json or something.

So this works. I am going to clean my repository space up yet again, delete my fork here, refork and clone, and see if I can't get this nipped in the bud.
